libraries/botbuilder-dialogs-adaptive-runtime-integration-azure-functions/src/index.ts

```diff
@@ -45,13 +46,17 @@ export function makeTriggers(
     runtimeServices: () => Promise<[ServiceCollection, Configuration]>,
     applicationRoot: string
 ): Record<string, AzureFunction> {
-    const instances = memoize(async () => {
-        const [services] = await runtimeServices();
-        return services.mustMakeInstances<{
+    const build = memoize(async () => {
+        const [services, configuration] = await runtimeServices();
+
+        const instances = services.mustMakeInstances<{
             adapter: BotFrameworkAdapter;
             bot: ActivityHandlerBase;
             channelServiceHandler: ChannelServiceHandler;
-        }>('adapter', 'bot', 'channelServiceHandler');
+            customAdapters: Map<string, BotFrameworkAdapter>;
```

Comment:
We’ll need to change these types in the runtime code to BotAdapter at some point 😉

Reply:
I think they should really be typed as `BotFrameworkHttpAdapter`.